spec/features/managing_employees_spec.rb

```diff
@@ -80,6 +122,35 @@
           expect(page).to have_content service_name
         end
 
+        scenario 'it check the inputs' do
+          click_link 'Change'
+          expect(page).to have_content 'Edit an employee'
+          expect(page).to have_selector("input[value=#{surname}]")
+          expect(page).to have_selector("input[value=#{forenames}]")
+          expect(find_field('Home address').value).to eq address
+          expect(find_field('Home postcode').value).to eq postcode
+          expect(find_field('Date of birth').value).to eq dob.strftime
+          expect(find_field('employee_employed_from').value).to eq employed_from.strftime
+          page.has_select?(:employee_service_id, selected: service_name)
```

Comment:
Looks like this is missing an `expect` - out of curiosity, if you changed this so the test should fail, e.g. by doing...

```
page.has_select?(:employee_service_id, selected: 'definitely not a service name or id')
```

...does the test still pass?

Reply:
You were right, it was passing. I've fixed that one :D